scss/_buttons.scss

```diff
@@ -3,15 +3,15 @@
 //
 
 .btn {
-  display: inline-block;
+  display: inline-flex;
+  align-items: center;
   font-family: $btn-font-family;
   font-weight: $btn-font-weight;
   line-height: $btn-line-height;
   color: $body-color;
   text-align: center;
   text-decoration: if($link-decoration == none, null, none);
   white-space: $btn-white-space;
-  vertical-align: middle;
```

Comment:
`vertical-align` applies to aligning the button to its parent, not aligning its content.